app/Http/Controllers/Multiplayer/RoomsController.php

```diff
@@ -18,6 +18,21 @@ public function __construct()
         $this->middleware('require-scopes:public', ['only' => ['index', 'leaderboard', 'show']]);
     }
 
+    /**
+     * Get Multiplayer Rooms
+     *
+     * Returns a list of multiplayer rooms.
+     *
+     * ---
```

Comment:
should only be here if there's response half of the docs otherwise this just adds a random line